cli/src/main.rs

```diff
@@ -6,7 +6,11 @@ use base64;
 use clap::{crate_name, crate_version, App, AppSettings, SubCommand};
 use serde_json::json;
 
-use anycloud::deploy::{client_error, get_config, info, new, terminate, upgrade, ALAN_VERSION};
+use anycloud::deploy::{
+  add_cred, add_deploy_config, client_error, edit_cred, edit_deploy_config, get_config, info,
+  list_creds, list_deploy_configs, new, remove_cred, remove_deploy_config, terminate, upgrade,
+  ALAN_VERSION,
+};
```

Comment:
I'm going to break this out into separate files in a follow up PR, but want to minimize merge conflicts with outstanding PRs for now